Assets/Scripts/Navigation/NavigationController.cs

```diff
@@ -31,8 +31,8 @@ public class Step
         private void Start()
         {
             UpdateGuide();
-            mapManager.SwitchMap(DataManager.Instance.CurrentMap);
-            meshRend = Steps[DataManager.Instance.CurrentStep].line.GetComponent<MeshRenderer>();
+            mapManager.SwitchMap(DataManager.CurrentMap);
```

Comment:
mapManager naming convention?

Reply:
not in the scope of this pr. navigation will be reworked on a later date